src/main/java/com/simpligility/maven/plugins/android/phase04processclasses/ProguardMojo.java

```diff
@@ -113,8 +112,8 @@
     private String[] parsedOptions;
 
     /**
-     * Path to the proguard jar and therefore version of proguard to be used. By default this will load the jar from
-     * the Android SDK install. Overriding it with an absolute path allows you to use a newer or custom proguard
+     * Path to the proguard jar and therefore version of proguard to be used. By default this will use Proguard 5.2.1
```

Comment:
Good point, updated.

On 16 May 2016 at 21:33, Malachi de AElfweald <notifications@github.com>
wrote:

> In
> src/main/java/com/simpligility/maven/plugins/android/phase04processclasses/ProguardMojo.java
> <https://github.com/simpligility/android-maven-plugin/pull/725#discussion_r63410938>
> :
>
> > @@ -113,8 +112,8 @@
> >      private String[] parsedOptions;
> >
> >      /**
> > -     * Path to the proguard jar and therefore version of proguard to be used. By default this will load the jar from
> > -     * the Android SDK install. Overriding it with an absolute path allows you to use a newer or custom proguard
> > +     * Path to the proguard jar and therefore version of proguard to be used. By default this will use Proguard 5.2.1
>
> Shouldn't specify the version in the comment, or the comment will have to
> be updated every time the dependency is. "use Proguard from the Central
> Repository" should be fine.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/simpligility/android-maven-plugin/pull/725/files/339e379c2d36df68e20846508b3c8dcac82e0817#r63410938>
>